Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml

```
@@ -0,0 +1,65 @@
+- type: entity
+  name: fox
```

```suggestion
  name: arctic fox
```
Wouldn't it make more sense to call it an "arctic" fox?

---

yes